### ISSUE TITLE
Clarify "inactivate"

### ISSUE THREAD

**Issue by michevnew** [MicrosoftDocs/microsoft-365-docs#1585]
In the following text:

"If you set up email forwarding, you cannot remove the license. Doing so will stop the email forwarding and inactivate the mailbox."

Can you please clarify what is the meaning of "inactivate" and/or rephrase it. If you are referring to the "inactive mailbox" functionality, make sure to explicitly mention that the mailbox needs to be on hold first. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f6245943-00b8-c47b-f739-731531d5d357
* Version Independent ID: a9ec7066-ed8c-7b77-1d87-8c50bf74ec6c
* Content: [Delete a user from your organization - Microsoft 365 admin](https://docs.microsoft.com/en-us/microsoft-365/admin/add-users/delete-a-user?view=o365-worldwide)
* Content Source: [microsoft-365/admin/add-users/delete-a-user.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/admin/add-users/delete-a-user.md)
* Service: **o365-administration**
* GitHub Login: @cmcatee-MSFT
* Microsoft Alias: **cmcatee**

**Comment by mypil** [comment]
@michevnew - Thank you for submitting feedback.  

PR #1666 has been created by @D3Design to fix and close this issue once the PR is merged.


Thank you and we appreciate your contribution to make the docs better!
